lib/europeana/page.rb

```diff
@@ -175,11 +175,19 @@ def menu_data
     # Support method for menu_data, remove upon refactor.
     #
     def navigation_global_primary_nav_collections_submenu_items
-      [
-        link_item('Art', URI.join(europeana_collections_url, 'collections/art'), is_current: false),
-        link_item('Fashion', URI.join(europeana_collections_url, 'collections/fashion'), is_current: false),
-        link_item('Music', URI.join(europeana_collections_url, 'collections/music'), is_current: false)
-      ]
+      collection_titles_and_slugs = {
+        '1914-1918' => 'world-war-I',
+        'Art' => 'art',
+        'Fashion' => 'fashion',
+        'Maps and Geography' => 'maps',
+        'Music' => 'music',
+        'Natural History' => 'natural-history',
+        'Photography' => 'photography',
+        'Sport' => 'sport'
+      }
+      collection_titles_and_slugs.map do |title, slug|
+        link_item(title, URI.join(europeana_collections_url, 'collections/', slug), is_current: false)
```

Comment:
Sorry, yes I realised that and had cancelled the comment (hence approved the PR) but it got through somehow.